sky/engine/core/script/dart_init.cc

```diff
@@ -114,6 +114,10 @@ static const char* kDartTraceStartupArgs[]{
     "--timeline_recorder=endless",
 };
 
+static const char* kVmCompleteTimelineArgs[]{
```

Comment:
Nice, done. ("--trace-startup" used not to set timeline recorder to endless back at the time of the first patchset)